### PR TITLE
Changed import_state to qml.qchem-level import

### PR DESCRIPTION
**Pull request by Chiffafox**
**Context:**

Docs change to switch import of `import_state` to `qml.qchem` level. 

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

**Comment by trbromley** [comment]
Thanks @Chiffafox! Please could you update the base of this PR to `v0.32.0-rc0`.

**Review by Jaybsoni — APPROVED** [review]
Looks good! Just one small comment about the change log 👍🏼

File: doc/releases/changelog-0.32.0.md [hunk 1]
```diff
@@ -371,6 +371,9 @@ array([False, False])
 * Input types and sources for external wavefunctions and operators for `qml.import_state` 
   and `qml.import_operator` are clarified. [(#4476)](https://github.com/PennyLaneAI/pennylane/pull/4476)
 
+* Changed `qml.import_state` to no longer import at PennyLane level: it is now to be imported 
+  as `qml.qchem.import_state`. [(#4505)](https://github.com/PennyLaneAI/pennylane/pull/4505)
+
```
[the remark this answers is not in the document]
Comment (soranjh) [reply doc/releases/changelog-0.32.0.md:376]: Great point but we are modifying the changelog [here](https://github.com/PennyLaneAI/pennylane/pull/4496).

[the remark this answers is not in the document]
Comment (Chiffafox) [reply doc/releases/changelog-0.32.0.md:376]: Thank you, sounds good! Will do.

**Pull request merged**